std/cells.act

```diff
@@ -389,8 +389,6 @@ export defcell LATCHINV (bool? CLK, D; bool! _q)
   [iskeeper=1;keeper=0] ~q<10> & ~CLK -> _q+
   [iskeeper=1] q<10> & _clk -> _q-
 
-  _q => q-
-
   }
   sizing { _clk{-2}; q{-1} }
```

Comment:
We can remove q{-1} from the sizing body since q is no longer driven by a gate here.